pandas/tests/frame/methods/test_diff.py

```diff
@@ -39,6 +39,38 @@ def test_diff(self, datetime_frame):
         expected = pd.DataFrame({"x": np.nan, "y": pd.Series(1), "z": pd.Series(1)})
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize("tz", [None, "UTC"])
```

Comment:
do we need a test with a TDI to start with? is Period supported?

Reply:
sufficient tests for Series? (assume so, just checking)

Reply:
yah, series goes through a whole different path

just added a test for timedelta, period goes through a different path